python/lsst/ts/vent/controller/run_dispatcher.py

```diff
@@ -0,0 +1,125 @@
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import argparse
+import asyncio
+import logging
+
+from .config import Config
+from .controller import Controller
+from .dispatcher import Dispatcher
+
+
+def parse_args() -> None:
+    parser = argparse.ArgumentParser(
+        prog="ts_atbuilding_vents",
+        description="Controls dome vents and exhaust fan via the RPi.",
+    )
+
+    parser.add_argument(
+        "--port",
+        type=int,
+        default=23,
+        help="Raspberry Pi port to listen for tcp commands.",
+    )
+    parser.add_argument(
+        "--modbus-host",
+        type=str,
+        default=Config.hostname,
+        help="Modbus hostname for the fan drive.",
+    )
+    parser.add_argument(
+        "--modbus-port",
+        type=int,
+        default=Config.port,
+        help="Modbus port number for the fan drive.",
+    )
+    parser.add_argument(
+        "--modbus-device-id",
+        type=int,
+        default=Config.device_id,
+        help="Modbus device ID for the fan drive.",
+    )
+
+    parser.add_argument(
+        "--max-freq",
+        type=float,
+        default=Config.max_freq,
+        help="Maximum fan frequency (Hz).",
+    )
+
+    parser.add_argument(
+        "--megaind_address",
+        type=int,
+        default=Config.megaind_bus,
+        help="Megaind card bus address.",
+    )
+
+    parser.add_argument(
+        "--megaind_stack_level",
+        type=int,
+        default=Config.megaind_stack,
+        help="Megaind card stack level.",
+    )
+
+    parser.add_argument(
+        "--sixteen_address",
+        type=int,
+        default=Config.sixteen_bus,
+        help="16inp card bus address.",
+    )
+
+    parser.add_argument(
+        "--sixteen_stack_level",
+        type=int,
+        default=Config.sixteen_stack,
+        help="16inp card stack level.",
+    )
+
+    return parser.parse_args()
+
+
+async def async_main():
+    args = parse_args()
+    log = logging.getLogger()
+
+    # Set up configuration
+    cfg = Config()
+    cfg.hostname = args.modbus_host
+    cfg.port = args.modbus_port
+    cfg.device_id = args.modbus_device_id
+    cfg.max_freq = args.max_freq
+    cfg.megaind_bus = args.megaind_address
+    cfg.megaind_stack = args.megaind_stack_level
+    cfg.sixteen_bus = args.sixteen_address
+    cfg.sixteen_stack = args.sixteen_stack_level
+
+    # Set up controller
+    controller = Controller(cfg, simulate=True)
+    await controller.connect()
+
+    # Set up dispatcher and attach controller
+    global dispatcher
+    dispatcher = Dispatcher(port=args.port, log=log, controller=controller)
+
+
+if __name__ == "__main__":
```

Comment:
Instead of doing this, make an executable in pyproject.toml, like we do with the CSCs. 


Check, for instance:

https://github.com/lsst-ts/ts_scriptqueue/blob/21b87c612388de817d9a11403667d221689120dd/pyproject.toml#L22

https://github.com/lsst-ts/ts_scriptqueue/blob/21b87c612388de817d9a11403667d221689120dd/python/lsst/ts/scriptqueue/script_queue.py#L512